### PR TITLE
rook/ceph: add regeneration safety

### PR DESCRIPTION
**Pull request by llamerada-jp**
Signed-off-by: Yuji Ito <llamerada.jp@gmail.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -178,6 +178,12 @@ jobs:
       - run: |
           cd team-management/template
           make validation
+      - run:
+          name: Check diff
+          command: |
+            make test-generate
+            diffs=$(git status -s)
+            if [ "$diffs" != "" ]; then printf "%s\n" "$diffs"; exit 1; fi
```
Comment (toshipp) [inline .circleci/config.yml:186]: Why don't you use `git diff --exit-code` instead?

Comment (toshipp) [inline .circleci/config.yml:186]: Why don't you use `git diff --exit-code` instead?

**Review by toshipp — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (llamerada-jp) [reply .circleci/config.yml]: refrected it

**Review by toshipp — DISMISSED** [review]

**Comment by toshipp** [comment]
@cupnes If you have no objection, please merge this.

**Review by cupnes — APPROVED** [review]

**Pull request merged**